cmd/agent/app/flags.go

```diff
@@ -22,69 +22,82 @@ package app
 
 import (
 	"flag"
-	"fmt"
 	"strings"
+
+	"github.com/spf13/viper"
+)
+
+const (
+	suffixWorkers             = "workers"
+	suffixServerQueueSize     = "server-queue-size"
+	suffixServerMaxPacketSize = "server-max-packet-size"
+	suffixServerHostPort      = "server-host-port"
+	collectorHostPort         = "collector.host-port"
+	httpServerHostPort        = "http-server.host-port"
+	discoveryMinPeers         = "discovery.min-peers"
 )
 
-// Bind binds the agent builder to command line options
-func (b *Builder) Bind(flags *flag.FlagSet) {
+// CreateFlags creates flag set for given builder
```

Comment:
Semantically, `this` would be better than `given`, as you are not being given a `Builder`. The doc could be `CreateFlags for this builder based on the application's parameters`